Generators/Generators.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
     runtimeOnly project(path: ':configs')
     runtimeOnly project(path: ':test-configs')
     runtimeOnly project(':extensions-kafka')
+    // dh-default.prop references io.deephaven.server.console.python.DebuggingInitializer
+    runtimeOnly project(':server')
```

Comment:
We should prefer to fix our "default" props, or have a different props file for Generators instead of adding this dependency IMO. (I've seen io.deephaven.server.console.python.DebuggingInitializer errors pop up in other places, so it may be worthwhile thinking about a more modular / dependency-sensitive approach to prop files instead of inheriting a single defaults file...)